packages/next/src/client/components/react-dev-overlay/internal/container/Errors.tsx

```diff
@@ -273,9 +278,34 @@ export function Errors({
               </small>
               <VersionStalenessInfo versionInfo={versionInfo} />
             </LeftRightDialogHeader>
-            <h1 id="nextjs__container_errors_label">
-              {isServerError ? 'Server Error' : 'Unhandled Runtime Error'}
-            </h1>
+
+            <div className="nextjs__container_errors__error_title">
+              <h1
+                id="nextjs__container_errors_label"
+                className="nextjs__container_errors_label"
+              >
+                {isServerError ? 'Server Error' : 'Unhandled Runtime Error'}
+              </h1>
+              <span>
+                {error.stack && (
+                  <CopyButton
+                    data-nextjs-data-runtime-error-copy-stack
+                    actionLabel="Copy error stack"
+                    successLabel="Copied"
+                    content={error.stack}
+                  />
+                )}
+                {debugInfo?.devtoolsFrontendUrl && (
```

Comment:
When is this not available? Can we disable the button instead if that's the case with a hint how to enable it? This would help discoverability.

Reply:
Update with a new design that we're always displaying this button in 2 cases: available and unavailable, with different icon and tips